src/qt/pivx/settings/forms/settingsfaqwidget.ui

```diff
@@ -0,0 +1,1365 @@
+                  <widget class="QWidget" name="widgetFaq8" native="true">
+                   <layout class="QHBoxLayout" name="horizontalpage_8" stretch="0,1">
+                    <item alignment="Qt::AlignTop">
+                     <widget class="QLabel" name="labelNumber8">
+                      <property name="minimumSize">
+                       <size>
+                        <width>24</width>
+                        <height>24</height>
+                       </size>
+                      </property>
+                      <property name="maximumSize">
+                       <size>
+                        <width>16777215</width>
+                        <height>24</height>
+                       </size>
+                      </property>
+                      <property name="text">
+                       <string>8</string>
+                      </property>
+                      <property name="alignment">
+                       <set>Qt::AlignCenter</set>
+                      </property>
+                     </widget>
+                    </item>
+                    <item>
+                     <layout class="QVBoxLayout" name="verticalLayoutpage_8">
+                      <property name="leftMargin">
+                       <number>20</number>
+                      </property>
+                      <item>
+                       <widget class="QLabel" name="labelSubtitle8">
+                        <property name="text">
+                         <string>What is PIVX</string>
+                        </property>
+                        <property name="wordWrap">
+                         <bool>true</bool>
+                        </property>
+                       </widget>
+                      </item>
+                      <item>
+                       <spacer name="verticalSpacer_Page24">
+                        <property name="orientation">
+                         <enum>Qt::Vertical</enum>
+                        </property>
+                        <property name="sizeType">
+                         <enum>QSizePolicy::Fixed</enum>
+                        </property>
+                        <property name="sizeHint" stdset="0">
+                         <size>
+                          <width>20</width>
+                          <height>30</height>
+                         </size>
+                        </property>
+                       </spacer>
+                      </item>
+                      <item>
+                       <widget class="QLabel" name="labelContent8">
+                        <property name="text">
+                         <string>We have support channels in most of our official chat groups for example #support in our Discord which can be joined by navigating to https://Discord.PIVX.org and if you prefer to submit a ticket, One can be submitted at &lt;a style='color: #b088ff' href='https://PIVX.FreshDesk.com '&gt;https://PIVX.FreshDesk.com&lt;/a&gt;
```

Comment:
grammar nit.  add comma: "most of our official chat groups, for example"   add period, remove and: "navigating to https://Discord.PIVX.org.  If you prefer"

Reply:
done 👍